### ISSUE TITLE
Allow to pass some extra namespace to debug.

### ISSUE THREAD

**Issue by freakpol** [daniel-duarte/flowed#25]
It would be convenient to have a way to add an extra namespace to the calls to `debug`, in order to better filter the logs using the DEBUG variable.

**Comment by daniel-duarte** [comment]
Added option to run flows with custom debug key (namespaces):

The optional parameter `options` can include the debug key as follows:

```
{
  debugKey: 'custom-scope'
}
```

This parameter can be specified when running flows in any of the forms:

```
FlowManager.run(flowSpec, params, expectedResults, resolvers, context, options)
FlowManager.runFromString(flowSpec, params, expectedResults, resolvers, context, options)
FlowManager.runFromFile(flowSpec, params, expectedResults, resolvers, context, options)
FlowManager.runFromUrl(flowSpec, params, expectedResults, resolvers, context, options)
```

**Comment by daniel-duarte** [comment]
Available in version `1.9.0`.
